include/func.sh

```diff
@@ -212,15 +212,35 @@ function wait_container_attached {
 }
 
 function wait_ns {
```

Comment:
Might be a good idea to have some comments on how to use this thing. (Currently it only supports one must have and one must not, and must have must be provided if must not is desired.)